ddtrace/_trace/tracer.py

```diff
@@ -180,9 +156,7 @@ def __init__(self) -> None:
             config._trace_compute_stats = False
         # Direct link to the appsec processor
         self._endpoint_call_counter_span_processor = EndpointCallCounterProcessor()
-        self._span_processors, self._appsec_processor = _default_span_processors_factory(
-            self._endpoint_call_counter_span_processor
-        )
+        self._span_processors = _default_span_processors_factory(self._endpoint_call_counter_span_processor)
```

Comment:
🤩  Awesome! Hopefully we can have a refactor from @DataDog/profiling-python too to move the endpoint call counter processor out as well at some point!